src/jquery/css.js

```diff
@@ -107,10 +107,15 @@ jQuery.fn.css = function( name, value ) {
 		} );
 	}
 	if ( typeof value === "number" ) {
-		camelName = camelCase( name );
-		if ( !isAutoPx( camelName ) && !jQuery.cssNumber[ camelName ] ) {
+		if ( typeof name === "object" ) {
 			migrateWarn( "Number-typed values are deprecated for jQuery.fn.css( \"" +
-				name + "\", value )" );
+					JSON.stringify( name ) + "\", value )" );
```

Comment:
If `name` is an object then `value` is ignored in `css` code so we shouldn't warn.

I think the bug is that we're not returning `this` in the `if` in line 104 which should already handle an object parameter. Can you try that?

You'll need to modify the test so that it mostly just asserts this doesn't crash but does its thing, just ignoring the second parameter.